docs/design/liqonet_routeOperator.md

```diff
@@ -0,0 +1,18 @@
+# Route-Operator
+## Overview
+The Route-Operator runs as a DaemonSet on all Kubernetes nodes and coordinates the setup of all the routes that allow each local pod/node to communicate with the pods of the peering cluster, through the elected gateway node.
```

Comment:
How is the gateway node elected?
Can the operator influence this election?

Reply:
the node is selected at install time labelling the node where the tunneloperator will run.